assets/base/i18n/en.json

```diff
@@ -2276,6 +2278,13 @@
   "revoke_user_sessions_modal.desc": "This action revokes all sessions for {username}. They will be logged out from all devices. Are you sure you want to revoke all sessions for {username}?",
   "revoke_user_sessions_modal.revoke": "Revoke",
   "revoke_user_sessions_modal.title": "Revoke Sessions for {username}",
+  "rename_channel.mobile.display_name_required": "Channel name is required",
```

Comment:
same with all this it should be `mobile.`whatever , once more `mobile.` should be the prefix